css-anchor-position-1/Overview.bs

```diff
@@ -160,10 +160,9 @@ a particular element is an <dfn>implicit anchor element</dfn>
 for a given positioned element.
 
 <p class=example>
-	For example, the Popover API allows a popover
-	to declare what element is anchoring it.
-	This makes the declared element the [=implicit anchor element=]
-	for the popover.
+	For example, the HTML spec defines an "anchor" attribute which allows an
```

Comment:
Maybe add a link to the HTML spec?

Reply:
How about linking it to https://whatpr.org/html/9144/dom.html#the-anchor-attribute for now, and then change it when the HTML PR is merged?

Reply:
done